third_party/chromium/media/base/starboard_utils.cc

```diff
@@ -396,6 +399,24 @@ SbMediaColorMetadata MediaToSbMediaColorMetadata(
 
   return sb_media_color_metadata;
 }
+
+int GetSbMediaVideoBufferBudgetMaximum() {
+  LOG(INFO) << "Checking for maximum video buffer budget.";
+  auto budget = SbMediaGetVideoBufferBudget(SbMediaVideoCodec::kSbMediaVideoCodecH264, 1920, 1080, 8);
+  if (SbMediaCanPlayMimeAndKeySystem(kVp9_8k_mime, "") == kSbMediaSupportTypeProbably) {
+    budget = SbMediaGetVideoBufferBudget(SbMediaVideoCodec::kSbMediaVideoCodecVp9, 7680, 4320, 12);
+    LOG(INFO) << "Device supports 8k.";
```

Comment:
LOG(INFO) << "Device supports 8k, using video budget " << budget;

So we can remove the log in line 404 and line 416.

Reply:
removed.